control/ipmitool.spec.in

```diff
@@ -1,10 +1,13 @@
+%define debug_package %{nil}
+
 Name:         ipmitool
 Summary:      ipmitool - Utility for IPMI control
 Version:      @VERSION@
 Release:      @RPM_RELEASE@%{?_distro:.%{_distro}}
 License:      BSD
 Group:        Utilities
 Packager:     Jim Mankovich <jmank@hp.com>
```

Comment:
Well, while we're at it, this spec file is just a template. I don't think it needs to contain the name of Jim because the resulting RPM package will definitely NOT be packaged by him. I think there must be some macro here like those `@VERSION@` and `@RPM_RELEASE@` above. I just don't know which.

Reply:
To make it simple, as the Packager info is not mandatory, I've put a commented example  : 
#Packager:    PackagerName <packager@example.com>